src/io/flutter/run/daemon/DaemonConsoleView.java

```diff
@@ -81,12 +81,6 @@ public void print(@NotNull String text, @NotNull ConsoleViewContentType contentT
       return;
     }
 
-    // "flutter test" prints this when using --start-paused.
-    // (We should probably change it to stop printing it in machine mode.)
-    if (trimmed.startsWith("You should first set appropriate breakpoints, then resume the test in the debugger.")) {
-      return;
-    }
-
```

Comment:
I suppose if the `--machine` flag fails we could try again without it, using a plain-text console. In that case, the debug button should be disabled altogether. Or maybe we could use the Flutter SDK version number, since we already have code for that.

More generally, it seems like we should have a generic way for the IDE to ask the Flutter SDK for a set of properties. This would let us turn on new features based on the presence of flags, and there are other things we could use this for, such as getting the Java classpath. (See https://github.com/flutter/flutter/pull/10802.)